instance/models/openedx_appserver.py

```diff
@@ -23,6 +23,7 @@
 from django.db import models
 from django.template import loader
 from django.utils.text import slugify
+from django_extensions.db.fields.json import JSONField
```

Comment:
@bradenmacdonald No, we can't, since it requires Postgres 9.5 or above, and our production server is 9.3.  I also had to update django_extensions to the latest version, since the `JSONField` in the version we used before was broken.